mlflow/pyfunc/__init__.py

```diff
@@ -828,6 +828,10 @@ def _create_model_downloading_tmp_dir(should_use_nfs):
 _MLFLOW_SERVER_OUTPUT_TAIL_LINES_TO_KEEP = 200
 
 
+def _infer_spark_datatype(spark, datatype: str):
```

Comment:
It's actually not infer schema because we explicitly pass the schema (in string). I would call it just `_parse_spark_datatype`

Reply:
```suggestion
def _parse_spark_datatype(spark, datatype: str):
```

Reply:
@HyukjinKwon Thanks for the suggestion!